content/hi/immutable-infrastructure.md

```diff
@@ -0,0 +1,23 @@
+---
+शीर्षक: अपरिवर्तनीय अवसंरचना (Immutable Infrastructure)
+status: Completed
+श्रेणी: संपत्ति
+टैग: ["बुनियादी ढांचा", "संपत्ति", ""]
+---
+
+अपरिवर्तनीय अवसंरचना का तात्पर्य कंप्यूटर अवसंरचना से है
+([वर्चुअल मशीन](/वर्चुअल-मशीन/, [कंटेनर](/कंटेनर/), नेटवर्क उपकरण)
+जिसे एक बार तैनात करने के बाद बदला नहीं जा सकता।
+इसे एक स्वचालित प्रक्रिया द्वारा लागू किया जा सकता है जो अनधिकृत परिवर्तनों को अधिलेखित कर देती है
+एक ऐसी प्रणाली के माध्यम से जो पहले स्थान पर परिवर्तन की अनुमति नहीं देगी।
+कंटेनर अपरिवर्तनीय बुनियादी ढांचे का एक अच्छा उदाहरण हैं
+क्योंकि कंटेनरों में लगातार परिवर्तन केवल कंटेनर का एक नया संस्करण 
+बनाके या मौजूदा कंटेनर को उसकी छवि से पुनः बनाके ही किए जा सकते हैं
```

Comment:
In this paragraph we are talking about the container image
छवि -> "Image(Picture)"
you can replace with `छवि` to `इमेज`